lib/mayaUsd/CMakeLists.txt

```diff
@@ -33,7 +33,6 @@ find_package(Boost COMPONENTS filesystem system REQUIRED)
 # -----------------------------------------------------------------------------
 target_compile_definitions(${PROJECT_NAME}
     PUBLIC
-        USD_VERSION_NUM=${USD_VERSION_NUM}
```

Comment:
So I assume by all the change below that pxr.h has the _#define PXR_VERSION xxx_ set correctly in all the USD versions we support.

Reply:
Yup, `PXR_VERSION` goes back at least as far as 20.02:
https://github.com/PixarAnimationStudios/USD/blob/v20.02/pxr/pxr.h.in#L33

It first appeared as a usable value in 19.11 thanks to @pmolodo:
https://github.com/PixarAnimationStudios/USD/commit/f75a464309daf895a317b1142861d39e47a47904